tools/bootstrapTools/bootstrap.targets

```diff
@@ -6,7 +6,9 @@
   </Target>
   
   <Target Name="Init" DependsOnTargets="GetLatestBuildTools">
-	  <Message Importance="high" Text="Build tools update complete." />
-    <Message Importance="high" Text="Please execute specific target to complete you build process (e.g. msbuild build.proj /t:RunTests /p:Scope=SDKs\Compute)"/>
+	  <Message Importance="high" Text="Build tools have been updated." />
```

Comment:
indention

Reply:
Thanks you, will fix it now

Reply:
I think my tab settings needs to be adjusted in VSCode, I again see no issues locally
![image](https://user-images.githubusercontent.com/20732208/58834638-ba010480-8608-11e9-80e5-844c4feebabe.png)

Reply:
you should also convert from tabs to spaces :)